controllers/users.js

```diff
@@ -558,6 +559,15 @@ const nonVerifiedDiscordUsers = async (req, res) => {
   return res.json(data);
 };
 
+const setInDiscordScript = async (req, res) => {
+  try {
+    await setInDiscordFalseScript();
+    return res.json({ message: "Successfully added the in_discord field to false for all users" });
```

Comment:
- Should we store this in a constant, than hardcoding the string?

Reply:
I do not feel so, because this API is just a 1-time thing, creating a constant didn't make sense.